core/src/main/java/ai/timefold/solver/core/config/util/ConfigUtils.java

```diff
@@ -372,12 +372,19 @@ public static List<Member> getAllMembers(Class<?> baseClass, Class<? extends Ann
                     .filter(field -> field.isAnnotationPresent(annotationClass) && !field.isSynthetic())
                     .sorted(alphabeticMemberComparator);
             var methodStream = Stream.of(clazz.getDeclaredMethods())
-                    .filter(method -> method.isAnnotationPresent(annotationClass) && !method.isSynthetic())
-                    .sorted(alphabeticMemberComparator);
+                    .filter(method -> method.isAnnotationPresent(annotationClass) && !method.isSynthetic());
```

Comment:
You can filter the entire concated stream at the end, avoiding this duplication.